pyiron_base/storage/datacontainer.py

```diff
@@ -920,13 +962,27 @@ def _force_load(self, recursive=True):
 
         # values are loaded from HDF once they are accessed via __getitem__, which is implicitly called by values()
         for v in self.values():
-            if recursive and isinstance(v, DataContainer):
+            if recursive and isinstance(v, type(self)):
```

Comment:
Similarly, should this be `(v, DataContainer)`? If my `MyDataContainer(DataContainer)` has `DataContainer` instances as items, don't those instances contain enough power to recurse like this?